Doc.html/alert.html

```diff
@@ -0,0 +1,140 @@
+            <li class="list-item" onclick="handleListItemClick()">Badge</li>
+            <li class="list-item" onclick="handleListItemClick()">Button</li>
+            <li class="list-item" onclick="handleListItemClick()">Cards</li>
+            <li class="list-item" onclick="handleListItemClick()">Input</li>
+            <li class="list-item" onclick="handleListItemClick()">Image</li>
+          </ul>
+        </aside>
+
+        <main class="main-container customThemeBox">
+
+          <div id="alerts">
+            <h1 class="title">Alerts</h1>
+            <p class="subtitle">
+              Alerts are used to grab user attention to show important message
+              on the screen without interrupting the user's flow.
+            </p>
+
+            <h2 class="small-title">Simple Alerts</h2>
+            <p class="para">
+              There are 5 types of simple alerts available to use as per the
+              situation like success, warning, info, error, custom. <br />
+              To use alert add common class (e.g., .alert) and add
+              second class as per the message you want to give user, for example
+              to give error alert message use (e.g., class = "alert
+              alert-error").
+            </p>
+
+            <div class="alert-container">
+              <div class="alert alert-success">
+                <div class="alert-content">
+                  <img class="alert-icon" src="../Components/icons/checked.png" />
+                  <h3 class="bold-normal">A success alert example!!!!</h3>
+                </div>
+              </div>
+
+              <div class="alert alert-warning">
+                <div class="alert-content">
+                  <img class="alert-icon" src="../Components/icons/warning.png" />
+                  <h3 class="bold-normal">A warning alert example!!!</h3>
+                </div>
+              </div>
+
+              <div class="alert alert-info">
+                <div class="alert-content">
+                  <img class="alert-icon" src="../Components/icons/information.png" />
+                  <h3 class="bold-normal">A information alert example!!!</h3>
+                </div>
+              </div>
+              <div class="alert alert-error">
+                <div class="alert-content">
+                  <img class="alert-icon" src="../Components/icons/error.png" />
+                  <h3 class="bold-normal">A error alert example!!!</h3>
+                </div>
+              </div>
+
+              <div class="alert alert-custom-msg">
+                <div class="alert-content">
+                  <img class="alert-icon" src="../Components/icons/message.png" />
+                  <h3 class="bold-normal">A custom message alert example!!! </h3>
+                </div>
+              </div>
+            </div>
+
+            <pre class="code-container"><code class="language-html">
+&lt;div clbss="alert alert-success"&gt;
```

Comment:
one suggestion, install prettier so that your code looks better and it indents on its own making code much prettier

You can look for prettier code formatter extension in your vs code extensions section.

Reply:
I have prettier , but yu have to understand that inside pre and code tag if I add indentation , that also get reflected in the code snippets, and that I dont want